AppHub/AppHub/AHBuild.m

```diff
@@ -58,13 +60,16 @@ - (BOOL)isDefaultBuild
 
 - (NSDictionary *)dictionaryValue
 {
-    return @{
+    NSMutableDictionary *dict = [@{
         AHExportedBuildDataBuildIDKey: _identifier,
         AHExportedBuildDataNameKey: _name,
         AHExportedBuildDataDescriptionKey: _buildDescription,
-        AHExportedBuildDataCreatedAtKey: @(_creationDate.timeIntervalSince1970 * 1000.0),
```

Comment:
nit: we could use a ternary operator here to avoid having to use the mutable dictionary